plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -62,7 +62,7 @@ type authorizingVisitor struct {
 func (v *authorizingVisitor) visit(source fmt.Stringer, rule *rbac.PolicyRule, err error) bool {
 	if rule != nil && RuleAllows(v.requestAttributes, rule) {
 		v.allowed = true
-		v.reason = fmt.Sprintf("allowed by %s", source.String())
+		v.reason = fmt.Sprintf("RBAC: allowed by %s", source.String())
```

Comment:
Reason is also set on [line 123](https://github.com/CaoShuFeng/kubernetes/blob/062f8130fd83e49b0b677b35a9f815ede85d68be/plugin/pkg/auth/authorizer/rbac/rbac.go#L123). 

I'd prefer to just tack on the `RBAC:` label in the authorize method.

Reply:
> Reason is also set on line 123.

Done.

Reply:
> I'd prefer to just tack on the RBAC: label in the authorize method.

This requires checking "len(reason) == 0"?
It makes the code a little bit more complex...